client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigationMetadata/InvestigationMetadata.tsx

```diff
@@ -7,16 +7,17 @@ import InvestigationMetaData from 'models/InvestigationMetadata';
 import InfoItem from '../InfoItem';
 import useStyles from './InvestigationMetadataStyles';
 
+const noStartDate = 'אין תאריך';
+
 const InvestigationMetadata = (props: Props) => {
 
     const classes = useStyles();
     const { investigationMetaData } = props;
 
     return (
         <Paper className={classes.metadata}>
-            <InfoItem testId='investigationStartDate' name='תאריך תחילת החקירה' value={
-                format(new Date(investigationMetaData.startTime), 'dd/MM/yyyy')
-            }
+            <InfoItem testId='investigationStartDate' name='תאריך תחילת החקירה'
+                value={investigationMetaData.startTime ? format(new Date(investigationMetaData.startTime), 'dd/MM/yyyy') : noStartDate}
```

Comment:
You can use format in Utils/Date utils

Reply:
I want the message to be no start date instead of no data